lib/guardian/bookmark_guardian.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module BookmarkGuardian
+  def can_delete_bookmark?(bookmark)
```

Comment:
Do we want staff/admins to be able to delete another user's bookmarks?

Reply:
Hmm good question...now that the Bookmarks link takes you to /bookmarks, which only shows your own bookmarks, I don't think there is a situation in which the admin could even _see_ the user's bookmarks. I wouldn't think it would be necessary for admin to mess with them?